snippets/survival_analysis/Coxph_model/Coxph_survival_regression.Rmd

```diff
@@ -8,61 +8,90 @@ author: "Jana Fischer"
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-# Read in data
+# Read in survival data
 ```{r}
 library(data.table)
+library(survival)
+library(ggplot2)
 
-surv_dat = fread('/home/jana/Github/bbsnippets/data/survival_example_data.csv',header = T)
+surv_dat = fread('~/Github/bbsnippets/data/survival_example_data.csv',header = T)
```

Comment:
This is one option. Alternatively, you could use the full link: `https://github.com/BodenmillerGroup/bbsnippets/blob/survival_example_data/data/survival_example_data.csv` But after merging, the branch needs to be adjusted to `master`

Reply:
What about using  a relative link instead?
`../../../data/survival_example_data.csv'`

I know @plankter did not like that a lot as it can give problems with continuous integration (CI) - but as long as the file is not supposed to be run by CI, this should be also be a good solution.

Otherwise the URL definitely be a good solution as well.

Reply:
Yeah, sorry, that's what I meant to add. That's how I usually link to data in repos.